bitrise.yml

```diff
@@ -221,19 +223,16 @@ workflows:
             exec unzip "$derived_data"
             echo "show dir"
             ls
-    - script@1.1:
+    - xcode-test-without-building@0:
```

Comment:
I did not realized we had two workflows here still with scripts. If you change this, which is what we should do, we need to also change the 
`custom-test-results-export@0:` and use this instead: 
```
- deploy-to-bitrise-io@2.1.1:
        run_if: '{{getenv "RUN_UI_TESTS" | eq "Run_UI_Tests"}}'
        inputs:
        - pipeline_intermediate_files: '$BITRISE_XCRESULT_PATH:BITRISE_UITESTS_2_XCRESULT_PATH'
```
As it is implemented for the two workflows that work (Smoktest2 and Smoktest4)
Because the failure on Bitrise is about that step not able to find the results to post them on the Test Tab on Bitrise UI.

Reply:
Sounds good. I was thinking that using the same Bitrise steps to do the same thing allows us to maintain the code easier in the future. 😄